identity-chaining-draft.md

```diff
@@ -75,6 +74,18 @@ informative:
     - name: D. Fett
       org: Authlete
 
+  OAUTH2-PROTECTED-RESOURCE-METADATA:
```

Comment:
I think with kramdown, rather than repeating all this reference inf, you can have just `  I-D.ietf-oauth-resource-metadata:` here and use `{{I-D.ietf-oauth-resource-metadata}}` later in the content

Reply:
for example, the use of `I-D.ietf-core-coap` in https://raw.githubusercontent.com/cabo/kramdown-rfc/master/examples/draft-ietf-core-block-xx.mkd

Reply:
another example see the `display: HTTPSIG` in https://raw.githubusercontent.com/httpwg/http-extensions/main/archive/draft-ietf-httpbis-client-cert-field.md